### ISSUE TITLE
Proportionality: Chapter 3 Types of quality assurance diagram is not accessible

### ISSUE THREAD

**Issue by zarbspace** [best-practice-and-impact/aqua_book_revision#1]
Figure 1 in Ch3 uses a Red Amber Green colour gradient which is not accessible. The figure needs to be updated with an accessible background shading palette.

**Comment by zarbspace** [comment]
Updated the diagram to use an accessible colour gradient. Note that this can now be changed relatively easily. Source diagram is in Powerpoint.
